Tests/XCUITests/DatabaseFixtureTest.swift

```diff
@@ -32,29 +44,179 @@ class DatabaseFixtureTest: BaseTestCase {
+    func testPerfHistory200startUp() {
+        waitForTabsButton()
+        measure(metrics: [
+            XCTMemoryMetric(),
+            XCTClockMetric(), // to measure timeClock Mon
+            XCTCPUMetric(), // to measure cpu cycles
+            XCTStorageMetric(), // to measure storage consuming
+            XCTMemoryMetric()]) {
+                // activity measurement here
+                app.launch()
+                waitForTabsButton()
+        }
+    }
+
+    func testPerfHistory200openMenu() {
+        waitForTabsButton()
+        measure(metrics: [
+            XCTMemoryMetric(),
+            XCTClockMetric(), // to measure timeClock Mon
+            XCTCPUMetric(), // to measure cpu cycles
+            XCTStorageMetric(), // to measure storage consuming
+            XCTMemoryMetric()]) {
+                navigator.goto(LibraryPanel_History)
+                let historyList = app.tables["History List"]
+                waitForExistence(historyList, timeout: TIMEOUT_LONG)
+                let expectedCount = 202
+                XCTAssertEqual(historyList.cells.count, expectedCount, "Number of cells in 'History List' is not equal to \(expectedCount)")
+        }
+    }
+
+    func testPerfHistory300startUp() {
+        waitForTabsButton()
+        measure(metrics: [
+            XCTMemoryMetric(),
+            XCTClockMetric(), // to measure timeClock Mon
+            XCTCPUMetric(), // to measure cpu cycles
+            XCTStorageMetric(), // to measure storage consuming
+            XCTMemoryMetric()]) {
+                // activity measurement here
+                app.launch()
+                waitForTabsButton()
+        }
+    }
+
+    func testPerfHistory300openMenu() {
+        waitForTabsButton()
+        measure(metrics: [
+            XCTMemoryMetric(),
+            XCTClockMetric(), // to measure timeClock Mon
+            XCTCPUMetric(), // to measure cpu cycles
+            XCTStorageMetric(), // to measure storage consuming
+            XCTMemoryMetric()]) {
+                navigator.goto(LibraryPanel_History)
+                let historyList = app.tables["History List"]
+                waitForExistence(historyList, timeout: TIMEOUT_LONG)
+                let expectedCount = 303
+                XCTAssertEqual(historyList.cells.count, expectedCount, "Number of cells in 'History List' is not equal to \(expectedCount)")
+        }
+    }
+
+    func testPerfHistory400startUp() {
+        waitForTabsButton()
+        measure(metrics: [
+            XCTMemoryMetric(),
+            XCTClockMetric(), // to measure timeClock Mon
+            XCTCPUMetric(), // to measure cpu cycles
+            XCTStorageMetric(), // to measure storage consuming
+            XCTMemoryMetric()]) {
+                // activity measurement here
+                app.launch()
+                waitForTabsButton()
+        }
     }
 
-    func testHistoryDatabaseFixture() throws {
+    func testPerfHistory500openMenu() {
```

Comment:
We should not remove that test. Instead the perf test should be implemented in the PerformanceTest suite

Reply:
As we have also discussed as the test is now fails as there are only 101 entries, not 404